File: VendorResponseSystem/ClientApp/src/models/buyer-vendor-configuration.ts
export class BuyerVendorConfiguration {
  public constructor(buyercode: string, vendcode: string, frequency: number, vendorname: string, vendoremail: string, vendorcomments: string, nyxnotes: string, confirmeddeliverydate: boolean, confirmedquantity: boolean, confirmedprice: boolean, isactive: boolean, isdelete: boolean, confirmedtrackingno: boolean) {
    this.buyercode = buyercode;
    this.vendcode = vendcode;
    this.frequency = frequency;
    this.vendorname = vendorname;
    this.vendoremail = vendoremail;
    this.vendorcomments = vendorcomments;
    this.nyxnotes = nyxnotes;
    this.confirmeddeliverydate = confirmeddeliverydate;
    this.confirmedquantity = confirmedquantity;
    this.confirmedprice = confirmedprice;
    this.confirmedtrackingno = confirmedtrackingno;
    this.isactive = isactive;
    this.isdelete = isdelete;
    }
  buyercode! : string;
  vendcode!: string;
  frequency!: number;
  vendorname!: string;
  vendoremail!: string;
  vendorcomments!: string;
  nyxnotes!: string;
  confirmeddeliverydate!: boolean;
  confirmedquantity!: boolean;
  confirmedprice!: boolean;
  confirmedtrackingno!: boolean;
  isactive!: boolean;
  isdelete!: boolean;
}

File: VendorResponseSystem/DataModels/BuyerVendorConfiguration.cs
using System;
using System.Collections.Generic;

namespace VendorResponseSystem.DataModels;

public partial class BuyerVendorConfiguration
{
    public string Buyercode { get; set; } = null!;

    public string Vendcode { get; set; } = null!;

    public int Frequency { get; set; }
    public string? Vendorcomments { get; set; }
    public string? Nyxnotes { get; set; }

    public bool Confirmeddeliverydate { get; set; }

    public bool Confirmedquantity { get; set; }

    public bool Confirmedprice { get; set; }
    public bool Confirmedtrackingno { get; set; }

    public bool? Isactive { get; set; }

    public bool? Isdelete { get; set; }
}
